### PR TITLE
Completed js-objects | Mohamed Elmessiry

### PR DESCRIPTION
**Pull request by Mohamed-Elmessiry**
Screen Shot 2021-01-25 at 6 51 41 PM

### CODE REVIEW

File: javascript-objects/main.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const student = {
+  firstName: 'Mohamed',
+  lastName: 'Elmessiry',
+  age: 34
+};
+const fullname = (student.firstName) + ' ' + (student.lastName);
```
Comment (codingcodymiller) [inline javascript-objects/main.js:6]: Please remove the parentheses, they are unnecessary

File: javascript-objects/main.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+console.log('value of fullname: ' + ' ' + fullname);
+(student.livesInIrvine) = false;
```
Comment (codingcodymiller) [inline javascript-objects/main.js:8]: Please remove the parentheses, they are unnecessary

File: javascript-objects/main.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+(student.previousOccupation) = 'Financial Analyst';
```
Comment (codingcodymiller) [inline javascript-objects/main.js:9]: Please remove the parentheses, they are unnecessary

File: javascript-objects/main.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+console.log('value of student lives in Irvine:' + ' ' + student.livesInIrvine);
+console.log('value of previous occupation:' + ' ' + student.previousOccupation);
+console.log(student);
+
+const vehichle = {
```
Comment (codingcodymiller) [inline javascript-objects/main.js:14]: This should be `vehicle`

File: javascript-objects/main.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  make: 'mercedes',
+  model: 'gullwing',
+  year: '1957'
+};
+(vehichle.color) = 'silver';
```
Comment (codingcodymiller) [inline javascript-objects/main.js:19]: Please remove the parentheses, they are unnecessary

File: javascript-objects/main.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+(vehichle.isConvertible) = false;
```
Comment (codingcodymiller) [inline javascript-objects/main.js:20]: Please remove the parentheses, they are unnecessary

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Yo dude! Got a few changes for you, please see my notes:

File: javascript-objects/main.js (hunk 6 above)
Comment (Mohamed-Elmessiry) [reply javascript-objects/main.js]: Screen Shot 2021-01-26 at 11 14 22 AM

**Review by codingcodymiller — APPROVED** [review]
🎉

**Pull request merged**